src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -145,12 +145,15 @@ const ArrayItem: React.VFC<
       disabled={isDisabled}
       leftGlyph={<Icon glyph="Trash" />}
       data-cy="delete-item-button"
+      size="small"
```

Comment:
I'm not sure, but this change may have affected the button size in array items (maybe it will be handled by the button PR)

<img width="375" alt="Screen Shot 2022-11-30 at 2 14 51 PM" src="https://user-images.githubusercontent.com/47064971/204888030-92512f23-d06e-4ba3-af6f-fc9ba5c29a1b.png">

Reply:
Thanks for catching — I bet it was messed up by updating text-input, but I'm going to note that it should be fixed in EVG-18234 in case the button update changes things again.